TictacToe/TictacToe/HumanPlayer.cs

```diff
@@ -26,42 +26,29 @@ public override int MakeAMove(char[,] board)
 
         public override int ThinkAMove(char[,] board)
         {
-            int input = -1;
-            var isValid = false;
+            var inputTxt = Console.ReadLine();
+            int userNumber;
 
-            while (!isValid)
+            try
             {
-                var inputTxt = Console.ReadLine();
-                isValid = ValidateInput(inputTxt, out int inputNum);
+                userNumber = int.Parse(inputTxt);
 
-                if (isValid) input = inputNum;
+                if (userNumber > 9 || userNumber < 1)
+                {
+                    throw new NumberGreaterOrEqualToOneAndLessOrEqualNine("Please enter number in range from 1 to 9!");
+                }
             }
-
-            return input;
-
-        }
-
-        private bool ValidateInput(string input, out int inputNum)
-        {
-            inputNum = -1;
-
-            if (string.IsNullOrWhiteSpace(input))
+            catch (NumberGreaterOrEqualToOneAndLessOrEqualNine error)
             {
-                Console.WriteLine("Field can't be empty! Try again.");
-                return false;
+                Console.WriteLine($"{error.Message}");
```

Comment:
Console.WriteLine($"{error.Message}");
to
                Console.WriteLine(error.Message);